etl/steps/data/garden/education/2023-12-15/wittgenstein_center.meta.yml

```diff
@@ -0,0 +1,64 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Global Education
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+  title: Wittgenstein Center Population and Human Capital Projections
+tables:
+  wittgenstein_center_data:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    variables:
+      no_formal_education:
+        title: Share of population with no education
+        unit: '%'
```

Comment:
Both indicators have pretty much identical metadata. We could reduce repetition by moving it to `common` (I'm not saying we have to move everything, but things like `description_key` or `description_processing` would be better there.. assuming they're identical, I haven't read every word.)